topic_monitor/setup.py

```diff
@@ -1,17 +1,27 @@
-from ament_python.data_files import get_data_files
-from ament_python.script_dir import install_scripts_to_libexec
+import sys
+
 from setuptools import find_packages
 from setuptools import setup
 
 package_name = 'topic_monitor'
-data_files = get_data_files(package_name)
-install_scripts_to_libexec(package_name)
+
+# install scripts into libexec path
+if 'develop' in sys.argv:
+    sys.argv[sys.argv.index('develop') + 1:sys.argv.index('develop') + 1] = \
```

Comment:
It doesn't. But I don't think it needs to either. ament/ament_tools#156 removes passing those from the build tool. So if no custom location is passed the package installs it's scripts into libexec. If a custom location is passed it is being honored (imo better than being ignored/overridden as it was before). The Debian rules file will need to do the same as the build tool (pass a prefix but no script specific location) and the behavior should not change.